addons/viewport/src/components/Panel.jsx

```diff
@@ -0,0 +1,120 @@
+/* global document */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { viewports, defaultViewport, resetViewport } from './viewportInfo';
+
+import { SelectViewport } from './SelectViewport';
+import { RotateViewport } from './RotateViewport';
+
+import * as styles from './styles';
+
+const storybookIframe = 'storybook-preview-iframe';
+const containerStyles = {
+  padding: 15,
+  width: '100%',
+  boxSizing: 'border-box',
+  fontFamily:
```

Comment:
Do not change the fonts please, or if you have to, you should get them from the `theme`-file from `lib/components`. That way we can ensure it's the same everywhere.